examples/app-basic/cypress/integration/02-app-navigation.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { e2e } from '@grafana/e2e';
+import { testIds } from '../../src/components/testIds';
+
+const { pageOne, pageFour, pageTwo, pageThree } = e2e.getSelectors(testIds);
+const headerTitle = 'Basic App Plugin';
+
+describe('navigating app', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/a/grafana-basic-app/one');    
+  });
+
+  it('should render page one successfully', () => {
+    // wait for page to successfully render
+    pageOne.container().should('be.visible');
+    cy.get('h1').contains(headerTitle).should('be.visible');
+  });
+
+  it('should render full width example successfully', () => {
+    // wait for page to successfully render
+    pageOne.container().should('be.visible');
+    cy.get('h1').contains(headerTitle).should('be.visible');
+    
+    // navigating to page four with full width layout
+    pageOne.navigateToFour().click();
+    cy.get('body').find('h1', ).should('not.exist');
```

Comment:
Is this syntax correct?

```suggestion
    cy.get('body').find('h1').should('not.exist');
```

Reply:
Oops, my bad 🙈  Thanks for the ping.